### ISSUE TITLE
rename `key_slope` to `feat_lm` ?

### ISSUE THREAD

**Issue by njtierney** [njtierney/brolgar#92]
As suggested by @milesmcbain - this will make it easier to find, and is more in line with the other names.

**Comment by njtierney** [comment]
Things to consider

- Does this behave similar to other `feat` functions?
- Expectations for input

**Comment by MilesMcBain** [comment]
Yeah I was definitely looking for something similar to the other `feat` functions. It would be a little unusual in that it needs a formula.

Maybe the formula could default to `.var ~ index`?

Where `.var` is the column that is supplied via `features` api and `index` is the `tsibble` index. Can `tsibbles` have multi-column indexes?
